<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaAsignarEntrenamiento</class>
 <widget class="QMainWindow" name="VistaAsignarEntrenamiento">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>900</width><height>600</height></rect>
  </property>
  <property name="windowTitle">
   <string>POWERGYM - Asignar Entrenamiento</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* (tu stylesheet existente…) */
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <!-- Botón Volver -->
    <item>
     <widget class="QPushButton" name="btn_volver">
      <property name="text"><string>Volver</string></property>
     </widget>
    </item>

    <!-- Selección de atleta -->
    <item>
     <layout class="QHBoxLayout" name="layoutSeleccionAtleta">
      <item>
       <widget class="QLabel" name="labelSeleccionAtleta">
        <property name="text"><string>Atleta:</string></property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboAtleta"/>
      </item>
     </layout>
    </item>

    <!-- Campos de Sentadilla, Banca, Peso Muerto… igual que tenías -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
       <!-- … tus layoutSentadilla, layoutPressBanca, layoutPesoMuerto … -->
     </layout>
    </item>

    <!-- Fecha de entrenamiento -->
    <item>
     <widget class="QDateEdit" name="fechaEntrenamiento">
      <property name="calendarPopup"><bool>true</bool></property>
      <property name="displayFormat"><string>yyyy-MM-dd</string></property>
     </widget>
    </item>

    <!-- Botón Guardar -->
    <item>
     <widget class="QPushButton" name="botonGuardar">
      <property name="minimumHeight"><number>40</number></property>
      <property name="text"><string>Guardar Sesión</string></property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
